backfill.sql

```diff
@@ -181,8 +189,17 @@ BEGIN
             $$;
     END IF;
 
-    IF on_conflict_do_nothing THEN
+    IF on_conflict_action = 'DoNothing' THEN
         on_conflict_clause = 'ON CONFLICT DO NOTHING';
+    ELSEIF on_conflict_action = 'Update' THEN
+        on_conflict_clause = 'ON CONFLICT DO UPDATE SET ';
+        FOR on_conflict_index IN 1 .. ARRAY_LENGTH(update_action_columns, 1) LOOP
```

Comment:
I would just do this with `SELECT 'ON CONFLICT DO UPDATE SET ' || STRING_AGG(FORMAT('%1$I = EXCLUDED.%1$I', on_conflict_column), ', ')`